### ISSUE TITLE
Carousel arrow buttons fail accessibility test

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#419]
The right and left navigation buttons on the homepage carousel have identical alt tags respectively for their button states which the accessibility software is reporting as an issue.

>Using the same alt text on adjacent images results in screen readers stuttering as the same text is read out twice.

**Comment by anilnatha** [comment]
This issue is similar to #418 in that the Carousel is managed by a third party plugin, SmartSlider. This plugin does not provide the ability to modify the scrolling controls so that the left and right buttons aren't duplicated which is what is causing this issue.

Our options are to:

1. Change the carousel component
2. Or, ask the maintainers of SmartSlider to update their plugin to resolve the 508 issues, we

**Comment by oddes** [comment]
@anilnatha 
these are not ready for QA. Moving them back.

**Comment by anilnatha** [comment]
I reached out to the maintainer of this plugin and provided them the details in hopes they can someday fix this issue.

**Comment by anilnatha** [comment]
The maintainer of the plugin has uncovered an issue with SortSite that I was not aware of. SortSite appears to be generating false positives in situations where an element has a property of `display: none`. In those cases, a screen reader shouldn't be reading those elements as it's not being displayed. This issue can be closed as I have verified with other online accessibility tools that it is not being reported.
